ports/rp2/boards/NULLBITS_BIT_C_PRO/mpconfigboard.h

```diff
@@ -0,0 +1,9 @@
+// https://nullbits.co/bit-c-pro
+
+#define MICROPY_HW_BOARD_NAME       "nullbits Bit-C PRO"
+#define MICROPY_HW_FLASH_STORAGE_BYTES (PICO_FLASH_SIZE_BYTES - (1 * 512 * 1024)) // 512K reserved
```

Comment:
Do you want a bigger filesystem?  I'd suggest 3MiB (same as PIMORONI_PICOLIPO_4MB).

Reply:
With the current configuration, doesn't that leave more room for the filesystem? Thonny reports 3.5MB free.

Reply:
Oh, I misread this as leaving only 512k for the filesystem.

So with the current configuration you have 512k for the firmware, and 3.5MiB for the filesystem.

The firmware is currently about 310k, so there is a fair bit of space left to get up to 512k.  So it should be OK to leave this as-is (ie 3.5MiB filesystem).

Reply:
awesome, thanks for confirming! I wasn't 100% sure.